README.md

```diff
@@ -393,7 +408,8 @@ Options:
   -f, --force           Force registration (will erase old katello and puppet
                         certs)
   --add-domain          Automatically add the clients domain to Foreman
-  --remove              Instead of registering the machine to Foreman remove it
+  --puppet-noop         Configure Puppet agent to only run in noop mode
+  --remove              Instead of registring the machine to Foreman remove it
```

Comment:
The script itself had the typo in its `--help` output, and I inadvertently copy/pasted it.